apps/spruce/src/pages/waterfall/VersionLabel/index.tsx

```diff
@@ -93,13 +93,24 @@ export const VersionLabel: React.FC<Props> = ({
         trimMessage={trimMessage}
       >
         <strong>{author}</strong> &bull;{" "}
-        {jiraLinkify(message, jiraHost, () => {
-          sendEvent({
-            name: "Clicked commit label",
-            "commit.type": commitType,
-            link: "jira",
-          });
-        })}
+        {jiraLinkify(
+          message +
+            message +
+            message +
+            message +
+            message +
+            message +
+            message +
+            message,
```

Comment:
Hmm I don't think this is right
<img width="504" alt="image" src="https://github.com/user-attachments/assets/e6b60de2-b525-48a9-a1bb-0699297b7eec">